examples_tests/test_busybox.py

```diff
@@ -25,9 +25,12 @@ def test_busybox(self):
         self.build_snap(self.example_dir)
         self.install_snap(self.example_dir, 'busybox', '1.0')
         self.assert_command_in_snappy_testbed(
-            ['/snap/bin/busybox.touch', 'busybox.test'], '')
+            ['/snap/bin/busybox.touch',
+             'snap/busybox/$(ls snap/busybox/)/busybox.test'],
```

Comment:
I agree that it's weird, but it was the shortest way I found to get the dir.
The ~ doesn't hurt. I added that.
I think that quotes are not needed because of how python passes the args, but I don't know that much about bash to understand all the details.